### PR TITLE
Prepare a 0.1.1 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# fm 0.1.1 (2020-07-08)
+
+* Add `ignore_leading_whitespace` and `ignore_leading_whitespace` options. Both
```
Comment (vext01) [inline CHANGES.md]: The second backticks should be `ignore_trailing_whitespace`.

Comment (ltratt) [reply CHANGES.md]: Fixed in https://github.com/softdevteam/fm/pull/15/commits/d4c4643e512c42b23fc5e996c579050fd94ac1ed.

**Comment by ltratt** [comment]
Can I squash please? :)

**Comment by vext01** [comment]
Oops. Go on then :P

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**